docs/content/how-to/powervs/create-infra-powervs-separately.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Create IBMCloud PowerVS Infra resources separately
+---
+
+# Create IBMCloud PowerVS Infra resources separately
+
+The default behavior of the `hypershift create cluster powervs` command is to create cloud infrastructure
+along with the HostedCluster and apply it. 
+
+It is possible to create the cloud infrastructure portion separately so that the `hypershift create cluster powervs` command can just be used to create the HostedCluster spec and apply it.
+
+Created HostedCluster can be destroyed and can be reinstalled with the same infra created earlier. 
+
+In order to do this, you need to:
+1. Create the PowerVS infrastructure
+2. Create the cluster
+
+## Creating the PowerVS infra
+
```

Comment:
how do I setup ibm powerVS perms to create my infra?

Reply:
Added a section in How-To doc.